### DISCUSSION TITLE
Percentage display in save file selection

### DISCUSSION THREAD

**Discussion by P-Star7** [Wohlstand/TheXTech#388] [General]
This shows more precise completion of the episode for big episodes with 100< stars or exits, I am not sure if it should be 1 decimal place or 2.

**Comment by Wohlstand** [comment]
Mainly a result of incorrectness comes up from episodes theme selves as they weren't properly designed or simply given an incorrect stars number, etc.

**Comment by P-Star7** [comment]
> Mainly a result of incorrectness comes up from episodes theme selves as they weren't properly designed or simply given an incorrect stars number, etc.

No, this means more specific of percentage displayed beside the save file than 0-100 integers. The same calculation is still used and can be done with error on episode designer's role. This idea I made
https://github.com/Wohlstand/TheXTech/issues/98
also solves a common problem, where people do not use "SMB3 Bowser - Game Beaten" event due to dislike of default credits, and the percentage counter on the save file cannot reach 100. That will make TheXTech-made episodes even more precise of the save file percentage counter even if the designer does not want to use "SMB3 Bowser - Game Beaten" event.
